CHANGELOG.md

```diff
@@ -12,6 +12,41 @@ Bugfixes:
 
 Other improvements:
 
+## [v5.0.0](https://github.com/purescript/purescript-prelude/releases/tag/v4.1.0) - 2021-MONTH-DATE
+
+Breaking changes:
+- Support compiler version `v0.14.0`, and drop support for previous versions (#206)
+- `purescript-proxy` was ported to this repo (#230)
+- `purescript-generics-rep` was ported to this repo (#235)
+- Move the `Applicative` Superclass law from `Monad` to `Bind` and rename it
+to the `Apply` Superclass law (#229)
+
+New features:
+- Added `Bounded` instance for records (#208)
+- Added `Show` instances to `Data.Generics.Rep` types (#250)
+
+Bugfixes:
+- No longer use reference equality check in `Array`'s `Eq` instance because it breaks referential transparency (#187).
+- Fix Ring laws (#228)
+
+Documentation improvements:
+- Added `lift2` example using `Maybe` (#213)
+- Added `const` example (#214)
+- Added `power` example (#253)
+- Improve documentation in various things (#217)
+    - Clarify `Array`'s "do notation"
+    - Clarify purpose of `Monoid` and `Semigroup` newtypes
```

Comment:
I think having a bullet point labelled "improve documentation in various things" under a parent "Documentation improvements" is a bit redundant. How about:
```suggestion
- Clarify `Array`'s do notation and the purposes of `Monoid` and `Semigroup` newtypes (#217)
```